packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -563,11 +563,11 @@ function TableView<T extends object>(props: SpectrumTableProps<T>, ref: DOMRef<H
         <DragPreview ref={preview}>
           {() => {
             let item = state.collection.getItem(dragState.draggedKey);
-            let rowHeaderColumnKeys = state.collection.rowHeaderColumnKeys;
             let itemCount = dragState.draggingKeys.size;
             let maxWidth = bodyRef.current.getBoundingClientRect().width;
-            let children = state.collection.getChildren(item.key);
-            return <SpectrumDragPreview layout={layout} item={item} children={children} itemCount={itemCount} density={density} direction={direction} maxWidth={maxWidth} rowHeaderColumnKeys={rowHeaderColumnKeys} />;
+            let {height} = layout.getLayoutInfo(item.key).rect;
+            let itemText = state.collection.getTextValue(dragState.draggedKey);
```

Comment:
I thought this was supposed to join all isRowHeader's? It looks like it's only getting one of the two in the example I shared in the previous comment.

Reply:
o, yep, i'm not sure what i was looking at. might've been confused by how close together the text was